### ISSUE TITLE
Train_on_batch for hdf5 data

### ISSUE THREAD

**Issue by Rashmeet09** [fchollet/keras#5864]
I want to train my train.h5 file in batch mode using keras for 10 epochs and display accuracy after each epoch. How do I exactly write it?
